substrate/frame/multisig/src/benchmarking.rs

```diff
@@ -47,48 +45,59 @@ fn setup_multi<T: Config>(
 	Ok((signatories, Box::new(call)))
 }
 
-benchmarks! {
-	as_multi_threshold_1 {
-		// Transaction Length
-		let z in 0 .. 10_000;
+#[benchmarks]
+mod benchmarks {
+	use super::*;
+
+	// `z`: Transaction Length
+	#[benchmark]
+	fn as_multi_threshold_1(z: Linear<0, 10_000>) -> Result<(), BenchmarkError> {
 		let max_signatories = T::MaxSignatories::get().into();
 		let (mut signatories, _) = setup_multi::<T>(max_signatories, z)?;
-		let call: <T as Config>::RuntimeCall = frame_system::Call::<T>::remark {
-			remark: vec![0; z as usize]
-		}.into();
-		let call_hash = call.using_encoded(blake2_256);
-		let multi_account_id = Multisig::<T>::multi_account_id(&signatories, 1);
+		let call: <T as Config>::RuntimeCall =
+			frame_system::Call::<T>::remark { remark: vec![0; z as usize] }.into();
 		let caller = signatories.pop().ok_or("signatories should have len 2 or more")?;
 		// Whitelist caller account from further DB operations.
 		let caller_key = frame_system::Account::<T>::hashed_key_for(&caller);
-		frame_benchmarking::benchmarking::add_to_whitelist(caller_key.into());
-	}: _(RawOrigin::Signed(caller.clone()), signatories, Box::new(call))
-	verify {
+		add_to_whitelist(caller_key.into());
+
+		#[extrinsic_call]
+		_(RawOrigin::Signed(caller.clone()), signatories, Box::new(call));
+
 		// If the benchmark resolves, then the call was dispatched successfully.
+		Ok(())
 	}
 
-	as_multi_create {
-		// Signatories, need at least 2 total people
-		let s in 2 .. T::MaxSignatories::get();
-		// Transaction Length
-		let z in 0 .. 10_000;
+	// `z`: Transaction Length
+	// `s`: Signatories, need at least 2 people
```

Comment:
```suggestion
	/// `s`: Signatories, need at least 2 people
```
It should also be possible to put doc comments here.

Reply:
Changed all to doc comments in https://github.com/paritytech/polkadot-sdk/pull/5995/commits/d46029526908a15a9d76c738afc92168643cea1b